### PR TITLE
359 fe implement add tag functionality

### PR DESCRIPTION
**Pull request by petark7**
Implement AddTag component
- rename AddTags to TagManagementControls for clarity
- move ToastContainer from ContactForm to layout.tsx
- minor UI modifications for better UX

### CODE REVIEW

File: app/content-panel/tags/TagTable.tsx [hunk 1]
```diff
@@ -24,7 +23,7 @@ const TagTable: React.FC = () => {
   };
 
   const handleDelete = (id: string) => {
-    console.log('Delete tag', id);
+    setTags(tags.filter((tag) => tag.id !== id));
```
Comment (edichoska) [inline app/content-panel/tags/TagTable.tsx:26]: on delete we would call the method, and update the data in the table.
this is functionality for another ticket. remove it from here

**Review by edichoska — APPROVED** [review]
Once the auth is done, we can finish this one and merge it.

**Comment by petark7** [comment]
Conflicts resolved. Will create another branch for tags functionality and rebase this once it is ready.

**Comment by petark7** [comment]
Base branch changed to [feature/base-branch](https://github.com/learnhubmk/app/tree/feature/tags-base)

**Pull request merged**